.github/workflows/release.yml

```diff
@@ -35,6 +35,19 @@ jobs:
         if: ${{ success() }}
         uses : ffurrer2/extract-release-notes@v1
 
+      - name: Get version
+        id: get_version
+        if: ${{ success() }}
+        run: echo "version=${GITHUB_REF/refs\/tags\//}" >> $GITHUB_ENV
+
+      - name: Create zip for sdkman
+        if: ${{ success() }}
+        run: |
+          cd ktlint-cli/build/run
+          mkdir -p ktlint-${{ env.version }}/bin
+          cp ktlint ktlint-${{ env.version }}/bin
+          zip -rm ktlint-${{ env.version }}.zip ktlint-${{ env.version }}
```

Comment:
Doing this for https://github.com/sdkman/sdkman-cli/wiki/Well-formed-SDK-archives

Reply:
Please add this as a comment before line 47